src/fasttrips.cpp

```diff
@@ -17,19 +17,21 @@ _fasttrips_initialize_parameters(PyObject *self, PyObject *args)
 {
     double     time_window;
     double     bump_buffer;
+    double     pat_variance;
     int        stoch_pathset_size;
     double     stoch_dispersion;
     int        stoch_max_stop_process_count;
     int        transfer_fare_ignore_pf;
     int        transfer_fare_ignore_pe;
     int        max_num_paths;
     double     min_path_probability;
-    if (!PyArg_ParseTuple(args, "ddidiiiid", &time_window, &bump_buffer, &stoch_pathset_size, &stoch_dispersion, &stoch_max_stop_process_count,
+
+    if (!PyArg_ParseTuple(args, "dddidiiiid", &time_window, &bump_buffer, &pat_variance, &stoch_pathset_size, &stoch_dispersion, &stoch_max_stop_process_count,
                                              &transfer_fare_ignore_pf, &transfer_fare_ignore_pe,
                                              &max_num_paths, &min_path_probability)) {
         return NULL;
     }
-    pathfinder.initializeParameters(time_window, bump_buffer, stoch_pathset_size, stoch_dispersion, stoch_max_stop_process_count, 
+    pathfinder.initializeParameters(time_window, bump_buffer, pat_variance, stoch_pathset_size, stoch_dispersion, stoch_max_stop_process_count, 
```

Comment:
just a flag for here to change this if change PAT to be PAT/PDT

Reply:
[ditto to rest of cpp]

Reply:
Addressed this point in Python and C++.